src/Graphics/src/Graphics/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,40 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("iOSUnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Benchmarks.Droid")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Core.DeviceTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.Android")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.Maps.Android")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.iOS")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.Windows")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.Tizen")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery.Android")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Compatibility.ControlGallery.Tizen")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Core.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.DeviceTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Foldable")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.HotReload.Forms")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Xaml")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Xaml.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Core.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Core.Android.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.DualScreen.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Essentials")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Graphics.Skia")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Pages.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.Xaml.UnitTests")]
+[assembly: InternalsVisibleTo("Core.Benchmarks")]
+[assembly: InternalsVisibleTo("CommunityToolkit.Maui")]
+[assembly: InternalsVisibleTo("CommunityToolkit.Maui.Core")]
+[assembly: InternalsVisibleTo("CommunityToolkit.Maui.UnitTests")]
+[assembly: InternalsVisibleTo("CommunityToolkit.Maui.Markup")]
+[assembly: InternalsVisibleTo("CommunityToolkit.Maui.Markup.UnitTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.TestUtils.DeviceTests.Runners")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.TestUtils.DeviceTests")]
+[assembly: InternalsVisibleTo("Microsoft.Maui.DeviceTests.Shared")]
```

Comment:
Do you really need all of these? Or can you just put the one, new one?
```csharp
[assembly: InternalsVisibleTo("Microsoft.Maui")]
```

Reply:
I trimmed it down to only what appears to be the assemblies that use MathF

Reply:
For this one, I don't know if that is necessary? `Mathf.cs` is a 46-line file that is only needed for `netstandard2.0`?